src/tests/Microsoft.PowerFx.Core.Tests/IRTests/IRTranslatorTests.cs

```diff
@@ -11,9 +12,15 @@
 
 namespace Microsoft.PowerFx.Core.Tests
 {
-    public class IRTranslatorTests
+    public class IRTranslatorTests : PowerFxTest
     {
-        private readonly EnumStore _enumStore = new EnumStoreBuilder().WithDefaultEnums().Build();
+        public IRTranslatorTests()
+            : base()
+        {
+            _enumStore = new EnumStoreBuilder().WithDefaultEnums().Build();
```

Comment:
>_enumStore [](http://example.com/codeflow?start=12&length=10)

Is the move here to change order of operations and define this *after* the base ctor runs?